src/components/application_manager/test/commands/mobile/add_sub_menu_request_test.cc

```diff
@@ -109,6 +114,49 @@ TEST_F(AddSubMenuRequestTest, OnEvent_SUCCESS) {
   EXPECT_EQ(mobile_apis::Result::SUCCESS, kReceivedResult);
 }
 
+TEST_F(AddSubMenuRequestTest,
+       OnEvent_SubMenuWithIcon_NotExistsInFolder_ExpectInfo) {
+  // message will be send on hmi with icon that not exists in folder
+
+  (*command_msg_)[am::strings::msg_params][am::strings::menu_id] = kMenuId;
+  (*command_msg_)[am::strings::params][am::strings::connection_key] =
+      kConnectionKey;
+  (*command_msg_)[am::strings::msg_params][am::strings::sub_menu_icon]
+                 [am::strings::value] = "icon.png";
+
+  Event event(hmi_apis::FunctionID::UI_AddSubMenu);
```

Comment:
@VVeremjova Please move variables to the place before their first usage. In all test cases.